### PR TITLE
Update German translations

### PR DESCRIPTION
**Pull request by hupfis**
Explanation:  
Fixing 2 wrong translations in German locale file.

### CODE REVIEW

**Comment by hupfis** [comment]
I have read the CLA Document and I hereby sign the CLA

File: app/_locales/de/messages.json [hunk 1]
```diff
@@ -691,7 +691,7 @@
     "message": "Datenschutzrichtlinie"
   },
   "privateKeyWarning": {
-    "message": "Warnung: Niemals jemanden deinen Private Key mitteilen. Jeder der im Besitz deines Private Keys ist, kann jegliches Guthaben deines Accounts stehlen."
+    "message": "Warnung: Niemals jemandem deinen Private Key mitteilen. Jeder der im Besitz deines Private Keys ist, kann jegliches Guthaben deines Accounts stehlen."
```
Comment (darkwing) [inline app/_locales/de/messages.json:694]: Google Translate shows both as `someone`; what's the reason for this change?  I don't know German, simply curious.

File: app/_locales/de/messages.json [hunk 2]
```diff
@@ -958,7 +958,7 @@
     "message": "Hierdurch werden ein neues Wallet und eine mnemonische Phrase erzeugt"
   },
   "tips": {
-    "message": "Trinkgelder"
+    "message": "Tipps"
```
Comment (darkwing) [inline app/_locales/de/messages.json:961]: Google Translate confirms this as "tips"

Comment (hupfis) [reply app/_locales/de/messages.json:961]: Correct. "Trinkgelder" would be like "tips" that you give to the waiter/waitress

File: app/_locales/de/messages.json (hunk 1 above)
Comment (hupfis) [reply app/_locales/de/messages.json:694]: The verb "mitteilen" requires the "Dativ", a special grammatical version of "jemand". As a proof, the German dictionary Duden shows the usage of mitteilen with "jemandeM" rather than "jemanden": 
https://www.duden.de/rechtschreibung/mitteilen
Example from that page: > "jemandem etwas brieflich, telefonisch, per E-Mail, schonend, vertraulich mitteilen"

**Review by ryanml — APPROVED** [review]

**Review by darkwing — APPROVED** [review]

**Comment by ryanml** [comment]
Thanks for the contribution!
